x-pack/plugins/ml/common/types/modules.ts

```diff
@@ -5,7 +5,7 @@
  * 2.0.
  */
 
-import { SavedObjectAttributes } from '@kbn/core/public';
+import { SavedObjectAttributes } from '@kbn/core/types';
 import { Datafeed, Job } from './anomaly_detection_jobs';
 import { ErrorType } from '../util/errors';
```

Comment:
Nit: Looks like we can change all imports here to `import type` (we could also take care in a follow up).

Reply:
Took care of this in [`c9c30ef` (#136911)](https://github.com/elastic/kibana/pull/136911/commits/c9c30ef715cc1624b0b27f9e5aa5bc34f8125441)